### PR TITLE
corrected tests according magic numbers task

### PR DESCRIPTION
**Pull request by vkhalin**
Corrected tests to match magic numbers issue that I've created recently. 
Link - https://github.com/msenin94/mk-post-deployment-checks/issues/77

@izadorozhna @dkulykov @legan4ik @msenin94 please review

### CODE REVIEW

File: mk_verificator/tests/ntp_sync/config.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ---
 skipped_nodes: ['saml-global-01.mosci.jiocloud.com']
 skipped_group: []
+sec_gauge_positive: 30
```
Comment (legan4ik) [inline mk_verificator/tests/ntp_sync/config.yaml]: I don't think we need 2 parameters for this. One should be enough. Also I would suggest renaming it to "time_deviation" or something like this.

File: mk_verificator/tests/packet_checker/test_packet_checker.py [hunk 2]
```diff
@@ -4,13 +4,15 @@
...
+    "group",
     utils.get_groups(utils.get_configuration(__file__))
 )
 def test_check_package_versions(local_salt_client, group):
+    config = utils.get_configuration(__file__)
+
     output = local_salt_client.cmd(group, 'lowpkg.list_pkgs')
 
-    if len(output.keys()) < 2:
+    if len(output.keys()) < config["min_nodes"]:
```
Comment (legan4ik) [inline mk_verificator/tests/packet_checker/test_packet_checker.py]: Why do we need this? This is pretty straight-forward - if we have less than 2 nodes - do not check results. Also we can rephrase like "if it is only 1 node of a kind".

File: mk_verificator/tests/packet_checker/test_packet_checker.py [hunk 3]
```diff
@@ -34,21 +36,23 @@ def test_check_package_versions(local_salt_client, group):
             row.sort()
             row.insert(0, deb)
             pkts_data.append(row)
-    assert len(pkts_data) <= 1, \
+    assert len(pkts_data) <= config["pkgs_data_len"], \
```
Comment (legan4ik) [inline mk_verificator/tests/packet_checker/test_packet_checker.py]: Need to discuss.

File: mk_verificator/tests/packet_checker/test_packet_checker.py [hunk 4]
```diff
@@ -34,21 +36,23 @@ def test_check_package_versions(local_salt_client, group):
...
+    "group",
     utils.get_groups(utils.get_configuration(__file__))
 )
 def test_check_module_versions(local_salt_client, group):
+    config = utils.get_configuration(__file__)
+
     pre_check = local_salt_client.cmd(group, 'cmd.run',
                                       ['dpkg -l | grep "python-pip "'])
-    if pre_check.values().count('') > 0:
+    if pre_check.values().count('') > config["zero"]:
```
Comment (legan4ik) [inline mk_verificator/tests/packet_checker/test_packet_checker.py]: Not sure we need this.

File: mk_verificator/tests/packet_checker/test_packet_checker.py [hunk 5]
```diff
@@ -34,21 +36,23 @@ def test_check_package_versions(local_salt_client, group):
...
         pytest.skip("pip is not installed on one or more nodes")
-    if len(pre_check.keys()) < 2:
+    if len(pre_check.keys()) < config["min_nodes"]:
```
Comment (legan4ik) [inline mk_verificator/tests/packet_checker/test_packet_checker.py]: See above

File: mk_verificator/tests/packet_checker/test_packet_checker.py (hunk 3 above)
Comment (legan4ik) [inline mk_verificator/tests/packet_checker/test_packet_checker.py]: Let's rename to "skip_number" and add notice to output.

File: mk_verificator/tests/packet_checker/test_packet_checker.py [hunk 6]
```diff
@@ -71,6 +75,6 @@ def test_check_module_versions(local_salt_client, group):
                 row.append("No module")
         if row.count(row[1]) < len(nodes):
             pkts_data.append(row)
-    assert len(pkts_data) <= 1, \
+    assert len(pkts_data) <= config["pkgs_data_len"], \
```
Comment (legan4ik) [inline mk_verificator/tests/packet_checker/test_packet_checker.py]: Let's rename to "skip_number" and add notice to output.

File: mk_verificator/tests/service_checker/test_services.py [hunk 7]
```diff
@@ -4,13 +4,15 @@
...
+    "group",
     utils.get_groups(utils.get_configuration(__file__))
 )
 def test_check_services(local_salt_client, group):
+    config = utils.get_configuration(__file__)
+
     output = local_salt_client.cmd(group, 'service.get_all')
 
-    if len(output.keys()) < 2:
+    if len(output.keys()) < config["min_nodes"]:
```
Comment (legan4ik) [inline mk_verificator/tests/service_checker/test_services.py]: See above

File: mk_verificator/tests/service_checker/test_services.py [hunk 8]
```diff
@@ -31,6 +33,6 @@ def test_check_services(local_salt_client, group):
                 row.append("No service")
         if row.count(row[1]) < len(nodes):
             pkts_data.append(row)
-    assert len(pkts_data) <= 1, \
+    assert len(pkts_data) <= config["pkts_data_gauge"], \
```
Comment (legan4ik) [inline mk_verificator/tests/service_checker/test_services.py]: Let's rename to "skip_number" and add notice to output.

File: mk_verificator/tests/state_checker/test_states.py [hunk 9]
```diff
@@ -23,12 +25,12 @@ def test_check_nodes_state(local_salt_client, group):
...
                     result.append("Failed. Info: {}".format(values['comment']))
-        if result == []:
+        if not result:
             continue
         elif result not in errors.values():
             errors[node] = result
         else:
             errors[node] = "Same errors"
-    assert len(errors.keys()) <= 1, \
+    assert len(errors.keys()) <= config["errors_gauge"], \
```
Comment (legan4ik) [inline mk_verificator/tests/state_checker/test_states.py]: Let's rename to "skip_number" and add notice to output.

File: mk_verificator/tests/system/single_vip/test_single_vip.py [hunk 10]
```diff
@@ -8,18 +8,21 @@
...
     local_salt_client.cmd(group, 'saltutil.sync_all')
-    nodes_list = local_salt_client.cmd(group, 'grains.item', ['ipv4'])
+    nodes_list = local_salt_client.cmd(group, 'grains.item',
+                                       [config["protocol"]])
 
     ipv4_list = []
 
     for node in nodes_list:
-        ipv4_list.extend(nodes_list.get(node).get('ipv4'))
+        ipv4_list.extend(nodes_list.get(node).get(config["protocol"]))
```
Comment (legan4ik) [inline mk_verificator/tests/system/single_vip/test_single_vip.py]: Do we need this in config?

File: mk_verificator/tests/system/single_vip/test_single_vip.py [hunk 11]
```diff
@@ -8,18 +8,21 @@
...
 
     cnt = Counter(ipv4_list)
 
     for ip in cnt:
-        if ip == '127.0.0.1':
+        if ip == config["host_ip"]:
```
Comment (legan4ik) [inline mk_verificator/tests/system/single_vip/test_single_vip.py]: Do we need this in config?

**Comment by legan4ik** [comment]
Please correct your code according to comments above and commit it. Also, please move test_single_vip.py to tests folder (let's remove "system" directory). Thanks

**Comment by msenin94** [comment]
I have nothing to add, thanks Oleksii! ;-)

File: mk_verificator/tests/system/single_vip/test_single_vip.py (hunk 10 above)
Comment (michaelsenin) [inline mk_verificator/tests/system/single_vip/test_single_vip.py]: I suppose nope, at least if you wanna use not only ipv4 protocol then rename `ipv4_list` variable

**Pull request merged**